pkg/deploy/tls/tls_utils.go

```diff
@@ -248,6 +248,7 @@ func doRequestForTLSCrtChain(ctx *deploy.DeployContext, requestURL string, skipP
 	transport.TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
 	client := &http.Client{
 		Transport: transport,
+		Timeout:   time.Second * 10,
```

Comment:
It may be worth using a shorter timeout (3 or 5 seconds) -- from my experience, this check is done three times during Che Operator startup, which means ~30 seconds before the operator is ready.

Reply:
Agree. I will update it